src/lib/optimizer/strategy/stored_table_column_alignment_rule.hpp

```diff
@@ -4,22 +4,21 @@
 #include <vector>
 
 #include "abstract_rule.hpp"
-#include "logical_query_plan/stored_table_node.hpp"
 
 /*
- The PQP sub-plan memoization (see LQPTranslator::_operator_by_lqp_node) for StoredTableNodes is sensitive to the node's
- table name, set of pruned chunks and set of pruned columns. Consequently, for multiple nodes with the same table name,
- same pruned chunks and same pruned columns, only one GetTable operator is created and executed. In some queries, the
- ColumnPruningRule and ChunkPruningRule provide an LQP with multiple StoredTableNodes where the table names and sets of
- pruned chunks are equal but the sets of pruned columns are different. This leads to the creation and execution of
- different GetTable operators instead of only one operator. While the cost of the additional GetTable operators is
- small, this cascades into more missed reuse opportunities for following operators.
+ The PQP subplan deduplication (see LQPTranslator::_operator_by_lqp_node) for StoredTableNodes is sensitive to the
```

Comment:
I'm missing a part in the comment that explains the changes introduced by this PR.

Reply:
Appended this to the end of the Doc:
_Please note, that this rule aligns columns of StoredTableNodes across a given plan and also, across all of its
 subplans, including subqueries._